fov_processing_pipeline/data/labkey.py

```diff
@@ -0,0 +1,110 @@
+import pandas as pd
```

Comment:
It seems this function was added, but it must have already existed, because we originally pulled from labkey

Reply:
I have seen this code before in the FOV processing pipeline